workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/compute.py

```diff
@@ -487,7 +487,7 @@ def time_mean_biases_dycore(prognostic, verification, grid):
     @add_to_diags("physics")
     @diag_finalizer(f"diurnal_{mask_type}")
     @transform.apply("mask_to_sfc_type", mask_type)
-    @transform.apply("resample_time", "1H", time_slice=slice(24, -1), inner_join=True)
+    @transform.apply("resample_time", "1H", inner_join=True)
```

Comment:
the report failed in the e2e test without this since the prognostic run goes for only 8 hours so there were no diurnal cycle diagnostics. We could also make the report generation more robust to missing diagnostics. But this time slicing is pretty arbitrary and I think a holdover from some specific plots we wanted last spring.